src/test/java/com/ericsson/ei/flowtests/FlowTest.java

```diff
@@ -65,6 +65,7 @@
         "missedNotificationDataBaseName: FlowTest-missedNotifications",
         "rabbitmq.exchange.name: FlowTest-exchange",
         "rabbitmq.consumerName: FlowTestConsumer" })
+
```

Comment:
Is there any reason for why this file is in the pull request?

Reply:
reason is removed... extra line